plugin/trino-delta-lake/pom.xml

```diff
@@ -14,11 +14,6 @@
     <description>Trino - Delta Lake connector</description>
 
```

Comment:
btw, have you seen/tried https://github.com/aws/aws-sdk-java-v2/tree/master/v2-migration ?